geozero-cli/Cargo.toml

```diff
@@ -10,7 +10,7 @@ name = "geozero"
 path = "src/main.rs"
 
 [dependencies]
-geozero = "0.9.0"
+geozero = "0.9.4"
```

Comment:
We'll need to bump this again when geozero gets released to be sure the CLI is using a version with CSV support. Alternatively I can bump the geozero/Cargo.toml version in this PR if you'd prefer.